http-wrapper/src/server.rs

```diff
@@ -39,10 +50,15 @@ impl SessionStore {
 
     async fn store_session(&self, session: Session) -> Result<Option<String>, SessionError> {
         self.store
-            .store_data(
-                session.id().to_string(),
-                Some(Duration::from_secs(600)),
-                session.clone(),
+            .store_data(session.id().to_string(), session.clone())
+            .await?;
+        self.store
+            .store_metadata(
+                &session.id().to_string(),
+                &fdo_store::MetadataKey::Ttl,
+                &chrono::Duration::from_std(Duration::from_secs(600)).map_err(|_| {
```

Comment:
Let's maybe make a global constant for session ttl, so it's simple to see and change?